app/routes/routes.js

```diff
@@ -34,6 +34,9 @@ var routes = [
   // Claim Summary
   require('./first-time/eligibility/claim/claim-summary'),
 
+  // File Upload
```

Comment:
don't need comments/sections for each route, refactor to group bank-account-details/claim-summary/file-upload under one section beneath expenses